src/main/java/com/endava/utility/Main.java

```diff
@@ -0,0 +1,17 @@
+package com.endava.utility;
+import java.util.List;
+import com.google.common.collect.Lists;
+import static com.google.common.collect.Lists.newArrayList;
+
+public class Main {
+
+    public static void main(String[] args) {
+        //uses import com.google.common.collect.Lists;
+        List<String> list1 = Lists.newArrayList("jedan", "dva", "tri");
```

Comment:
Just to note you don't need List.newArrayList(...), you can just use newArrayList(...) 
Also comment on line 9 seems unnecessary to me

But these two things not a complain, they are just a different style.